cmd/otelcontribcol/go.mod

```diff
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.21
 
-toolchain go1.21.6
+toolchain go1.21.7
```

Comment:
Hmmm probably doesn't need this after https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31224.